guides/plugins/plugins/plugins-for-symfony-developers.md

```diff
@@ -1,2 +1,64 @@
 # Plugins for Symfony developers
 
+## Overview
+
+This guide serves as an entry point for developers, who are familiar with the concepts of `Symfony bundles`.
+
+## Prerequisites
+
+This guide handles some base concepts of Shopware plugins. 
+You may want to have a look at [plugin base guide](./plugin-base-guide.md) first.
+
+As this guide also references the functionality of Symfony bundles,
+you should have at least basic knowledge of it.
+You may want to have a look or refresh your knowledge with help of Symfony's [bundle documentation](https://symfony.com/doc/current/bundles.html).
+
+## Symfony bundles
+
+A bundle is the Symfony's preferred way to provide additional third-party features to any Symfony application.
+Those bundles are everywhere: Symfony even outsources many of its core features into external bundles.
+The template engine `Twig`, the `Security` bundle, the `WebProfiler`, 
+as well as many other third-party bundles can be installed on demand to extend your Symfony application in any way.
+The Bundle System is Symfony's way of providing an extendable framework with plugin capabilities.
+
+## Shopware plugins
+
+Shopware builds upon the `Symfony Bundle System` to extend its functionality even more.
+This allows the `Shopware Plugin System` to function as a traditional plugin system
+with features like plugin lifecycles and more.
+
+Whenever you create a Shopware plugin, you have to extend the `Shopware\Core\Framework\Plugin` class.
+If you investigate this class, you will see, that this class extends `Shopware\Core\Framework\Bundle`,
+which in turn extends the Symfony's `Bundle` class:
+
+{% code title="platform/src/Core/Framework/Bundle.php" %}
+```php
+<?php declare(strict_types=1);
```

Comment:
A while ago, I've built this slide to explain the inheritance hierarchy - maybe we can include that somehow as well?

![image](https://user-images.githubusercontent.com/7780750/111323910-3213af80-8662-11eb-9b7d-7825dda4b68c.png)

Reply:
Nice! :-O